packages/storybook-addon-center/src/index.tsx

```diff
@@ -8,6 +8,8 @@ export const withCenter = makeDecorator({
   name: ADDON_ID,
   parameterName: PARAM_KEY,
   wrapper: (storyFn, context, { parameters }) => {
+    if (parameters?.disabled) return storyFn(context)
```

Comment:
Why add a disabled param? Why not just not use Centered in your storybook setup?

Reply:
i wanted centered on some stories but wanted to disable the addon on only some stories. this is the general approach that i really should have added in the beginning